client/a8c-for-agencies/sections/marketplace/pressable-overview/plan-selection/style.scss

```
@@ -54,6 +54,12 @@
 	font-weight: 600;
 }
 
+.pressable-overview-plan-selection__details-card-header-comming-soon {
+	font-size: 1rem;
+	line-height: 1.4;
+	font-weight: 400;
```

I suggest using the the mixin we have here to stay consistent:

```suggestion
	@include a4a-font-heading-md($font-weight: 400);
```

We have bunch of mixins added here: https://github.com/Automattic/wp-calypso/blob/45ac13c0218d3524632db6f490506714daeed3f6/client/assets/stylesheets/shared/mixins/_a4a-typography.scss